cmd/gateway/azure/gateway-azure.go

```diff
@@ -685,10 +685,22 @@ func (a *azureObjects) GetObjectInfo(ctx context.Context, bucket, object string,
 		return objInfo, azureToObjectError(err, bucket, object)
 	}
 
+	etag := minio.ToS3ETag(blob.Properties.Etag)
+	switch {
+	case blob.Properties.ContentMD5 != "":
```

Comment:
This looks like a breaking change, since it seems that we are now generating new ETag which are equal to md5sum in put object, I think then that the etag/md5sum problem will be fixed with new uploaded objects

Reply:
It's not a breaking change.. it's simply a fix.

Reply:
> It's not a breaking change.

If I understood correctly, GetObjectInfo will return different ETag value for HEAD object when users update their Minio verion

Reply:
Yes but it's not something they are dependent on anyways, this simply addresses the fact that some applications are not working without this change.  HTTP reply wise its a breaking change yes but we have a broken implementation anyways so it is a fix for the broken implementation.